common/src/main/resources/assets/scannable/lang/en_us.json

```diff
@@ -23,7 +23,8 @@
   "item.scannable.entity_module": "Scanner Module: Living Entity",
   "item.scannable.entity_module.desc": "Detects configured living entities. Use on an entity to add it to the configuration. Use on air to manage configuration",
   "item.scannable.entity_module.list": "Configured entities:",
-
+  "item.scannable.chest_module": "Scanner Module: Chests",
+  "item.scannable.chest_module.desc": "Detects chests blocks (e.g. chests, barrels, shulker boxes).",
```

Comment:
Think this should read "Detects chest blocks" (i.e. no plural on the chest), no?  That said, given the list it picks up, a better description might be "Detects storage blocks"?